kong/db/migrations/core/017_300_to_310.lua

```diff
@@ -8,13 +8,115 @@ return {
             -- Do nothing, accept existing state
             END;
         $$;
+
+        create table if not exists "key_sets" (
```

Comment:
It's pretty minor, but would you mind adjusting the case of your SQL to match the rest of the migrations (i.e. `create table` => `CREATE TABLE`)?

Reply:
uppercase'd all-the-things